media/base/decoder_buffer.h

```diff
@@ -266,6 +266,7 @@ class MEDIA_EXPORT DecoderBuffer
   // If there's no data in this buffer, it represents end of stream.
 #if BUILDFLAG(USE_STARBOARD_MEDIA)
   bool end_of_stream() const { return !data_; }
+  void shrink_to(size_t size) { DCHECK_LE(size, size_); size_ = size; }
```

Comment:
There's two statements on this line. It's probably ok, but could you double check with the linter?

Reply:
Fixed.